Sharad/Question 3 on vector/link is here.txt

```diff
@@ -0,0 +1 @@
+https://www.hackerrank.com/challenges/vector-sort/problem
```

Comment:
Include this link in question.txt only. Don't make a separate file for it.

Reply:
done now check